compiler/x/codegen/X86Debug.cpp

```diff
@@ -620,12 +620,9 @@ TR_Debug::print(TR::FILE *pOutFile, TR::X86FenceInstruction  * instr)
          trfprintf(pOutFile, " External Absolute [");
       else
          trfprintf(pOutFile, " Relative [");
-
-      if (!_comp->getOption(TR_MaskAddresses))
-         {
          for (auto i = 0U; i < instr->getFenceNode()->getNumRelocations(); ++i)
             trfprintf(pOutFile," " POINTER_PRINTF_FORMAT, instr->getFenceNode()->getRelocationDestination(i));
```

Comment:
nitpick: indentation; only mentioning it as this is not part of the `else` above.

Reply:
Fixed in forced push.